algorithms/linfa-clustering/src/k_means/algorithm.rs

```diff
@@ -365,7 +365,7 @@ fn get_random_centroids<F: Float, D: Data<Elem = F>>(
 mod tests {
     use super::*;
     use approx::assert_abs_diff_eq;
-    use ndarray::{array, stack, Array, Array1, Array2, Axis};
+    use ndarray::{array, concatenate, Array, Array1, Array2, Axis};
```

Comment:
`concatenate` is really long, I don't know why they didn't go for the abbreviation `concat`